test/e2e/framework/test_context.go

```diff
@@ -258,6 +258,8 @@ type NodeTestContextType struct {
 	RestartKubelet bool
 	// ExtraEnvs is a map of environment names to values.
 	ExtraEnvs map[string]string
+	// StandaloneMode indicates whether the test is running kubelet in a standalone mode.
+	StandaloneMode bool
```

Comment:
NAK (I know, too late).

e2e_node options should get removed from TestContext, not added to it.

Reply:
it's never too late. Let's iterate on this

Reply:
Yes, that's why I commented after merging :smile: